js/Track/Model/File/BIGWIG.js

```diff
@@ -0,0 +1,19 @@
+Genoverse.Track.Model.File.BIGWIG = Genoverse.Track.Model.Graph.Bar.extend({
+
+  getData : function(chr, start, end){
+    var model = this;
+    var deferred = $.Deferred();
+
+    if(!this.url) this.data = new dallianceLib.BlobFetchable(this.track.dataFile);
+    else this.data = new dallianceLib.URLFetchable(this.url);
```

Comment:
Replace the above two lines with
```
this.bigwigFile = this.bigwigFile || this.url ? new dallianceLib.URLFetchable(this.url) : new dallianceLib.BlobFetchable(this.track.dataFile);
```
so that we don't make a new dallianceLib object every time getData is called